packages/storage/src/providers/s3/utils/resolveCredentials.ts

```diff
@@ -1,13 +1,14 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { AmplifyV6 } from '@aws-amplify/core';
+import { fetchAuthSession } from 'aws-amplify/auth';
```

Comment:
```suggestion
import { fetchAuthSession } from 'aws-amplify/core';
```

The storage package should NOT have dependency over `auth` package.

Reply:
Updated. Thanks